doc/releases/reef.rst

```diff
@@ -6,6 +6,417 @@ Reef is the 18th stable release of Ceph. It is named after the reef squid
+* client: fix leak of file handles (`pr#56122 <https://github.com/ceph/ceph/pull/56122>`_, Xavi Hernandez)
+* client: Fix return in removexattr for xattrs from `system.` namespace (`pr#55803 <https://github.com/ceph/ceph/pull/55803>`_, Anoop C S)
+* client: queue a delay cap flushing if there are ditry caps/snapcaps (`pr#54466 <https://github.com/ceph/ceph/pull/54466>`_, Xiubo Li)
+* client: readdir_r_cb: get rstat for dir only if using rbytes for size (`pr#53359 <https://github.com/ceph/ceph/pull/53359>`_, Pinghao Wu)
+* cmake/modules/BuildRocksDB.cmake: inherit parent's CMAKE_CXX_FLAGS (`pr#55502 <https://github.com/ceph/ceph/pull/55502>`_, Kefu Chai)
+* cmake: use or turn off liburing for rocksdb (`pr#54122 <https://github.com/ceph/ceph/pull/54122>`_, Casey Bodley, Patrick Donnelly)
+* common/options: Set LZ4 compression for bluestore RocksDB (`pr#55197 <https://github.com/ceph/ceph/pull/55197>`_, Mark Nelson)
+* common/weighted_shuffle: don't feed std::discrete_distribution with all-zero weights (`pr#55153 <https://github.com/ceph/ceph/pull/55153>`_, Radosław Zarzyński)
+* common: resolve config proxy deadlock using refcounted pointers (`pr#54373 <https://github.com/ceph/ceph/pull/54373>`_, Patrick Donnelly)
+* DaemonServer.cc: fix config show command for RGW daemons (`pr#55077 <https://github.com/ceph/ceph/pull/55077>`_, Aishwarya Mathuria)
+* debian: add ceph-exporter package (`pr#56541 <https://github.com/ceph/ceph/pull/56541>`_, Shinya Hayashi)
+* debian: add missing bcrypt to ceph-mgr .requires to fix resulting package dependencies (`pr#54662 <https://github.com/ceph/ceph/pull/54662>`_, Thomas Lamprecht)
+* doc/architecture.rst - fix typo (`pr#55384 <https://github.com/ceph/ceph/pull/55384>`_, Zac Dover)
+* doc/architecture.rst: improve rados definition (`pr#55343 <https://github.com/ceph/ceph/pull/55343>`_, Zac Dover)
+* doc/architecture: correct typo (`pr#56012 <https://github.com/ceph/ceph/pull/56012>`_, Zac Dover)
+* doc/architecture: improve some paragraphs (`pr#55399 <https://github.com/ceph/ceph/pull/55399>`_, Zac Dover)
+* doc/architecture: remove pleonasm (`pr#55933 <https://github.com/ceph/ceph/pull/55933>`_, Zac Dover)
+* doc/cephadm - edit t11ing (`pr#55482 <https://github.com/ceph/ceph/pull/55482>`_, Zac Dover)
+* doc/cephadm/services: Improve monitoring.rst (`pr#56290 <https://github.com/ceph/ceph/pull/56290>`_, Anthony D'Atri)
+* doc/cephadm: correct nfs config pool name (`pr#55603 <https://github.com/ceph/ceph/pull/55603>`_, Zac Dover)
+* doc/cephadm: improve host-management.rst (`pr#56111 <https://github.com/ceph/ceph/pull/56111>`_, Anthony D'Atri)
+* doc/cephadm: Improve multiple files (`pr#56130 <https://github.com/ceph/ceph/pull/56130>`_, Anthony D'Atri)
+* doc/cephfs/client-auth.rst: correct ``fs authorize cephfs1 /dir1 clie… (`pr#55246 <https://github.com/ceph/ceph/pull/55246>`_, 叶海丰)
```

Comment:
I think this needs to become
```
doc/cephfs/client-auth.rst: correct ``fs authorize cephfs1 /dir1 client.x rw`` (`pr#55246 <https://github.com/ceph/ceph/pull/55246>`_, 叶海丰)
```
to fix the missing closing double backtick which breaks the doc build:
```
Check warning on line 85 in doc/releases/reef.rst
CephaCheck / ceph-pr-docs

sphinx

Inline literal start-string without end-string.
```

Reply:
I think I got it, thx

Reply:
I'd suggest getting rid of the ellipsis (i.e. those three dots).  GitHub truncated the title of that PR, but there is no reason to carry that over and not include the actual command in the release notes.
```
``fs authorize cephfs1 /dir1 client.x rw``
```

Reply:
removed and also double quotes

Reply:
The ellipsis itself wasn't the issue, the truncated command was -- "clie" is not a word.  In the original comment, I provided a line to cut-and-paste:
```
doc/cephfs/client-auth.rst: correct ``fs authorize cephfs1 /dir1 client.x rw`` (`pr#55246 <https://github.com/ceph/ceph/pull/55246>`_, 叶海丰)
```

Reply:
done, added * at the begining